src/honey_badger/epoch_state.rs

```diff
@@ -116,6 +116,8 @@ pub struct EpochState<C, N: Rand> {
     subset: SubsetState<N>,
     /// The status of threshold decryption, by proposer.
     decryption: BTreeMap<N, DecryptionState<N>>,
+    /// Nodes found so far in `Subset` output.
+    nodes_found_so_far: BTreeSet<N>,
```

Comment:
Maybe `accepted_proposers`?
This is the set of IDs of those nodes whose contributions we already know have been voted into the next batch.

Reply:
Done